dist/index.html

```diff
@@ -219,17 +219,20 @@ <h2>Loaded Chocolate Chip Pudding Cookie Cups</h2>
         </div>
       </article>
     </section>
-    <section id="clickedRecipe" class="hidden">
-      <section class="recipe-details" id="recipeLeftSide">
-        <div id="recipeName"></div>
-        <div id="recipeInstructions"></div>
-      </section>
-      <section class="recipe-details" id="recipeRightSide">
-        <img id="closeRecipe" src="delete.png"></img>
-        <h3>Ingredients</h3>
-        <div id="ingredientsList"></div>
+    <div class="'flex-container"> 
```

Comment:
can the flex-container be named something specific to what it contains?

Reply:
Oh, actually I need to remove this since we are going to tackle centering the recipe card on a different ticket! Thanks for the callout.

Reply:
Ok done now!